fontforgeexe/startui.c

```diff
@@ -304,17 +320,6 @@ void DelayEvent(void (*func)(void *), void *data) {
     GDrawRequestTimer(splashw,100,0,info);
 }
 
-static void DoDelayedEvents(GEvent *event) {
```

Comment:
`DoDelayedEvents()` is the match for `DelayEvent()`, in the sense that you call the latter to initiate some event at a later time and the former does the initiation. These changes don't remove `DelayEvent()` and it's called in four other files. Therefore with these changes some events that were supposed to happen won't. I've confirmed one of those: At the present branch state if you open a Metrics Window and then choose "Element -> Font Info ..." the window doesn't open.

Probably easiest to just put this function and the else clause back.